test_scripts/Policies/HMI_PTU/016_Successful_PTU_after_failed_retry_parallel_trigger_ext.lua

```diff
@@ -0,0 +1,125 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Currently this test case does not pass. 
```
[01:18:22,622] Unsuccessful_PTU_via_a_mobile_device                                                  [FAIL] (60024 ms)
 HMI notification SDL.OnStatusUpdate: Timeout: HMI notification SDL.OnStatusUpdate: Timeout expired
 HMI notification SDL.OnStatusUpdate: ValidIf: 
params.status: expected: UPDATING, actual value: UPDATE_NEEDED
params.status: expected: UPDATE_NEEDED, actual value: UPDATING
params.status: expected: UPDATING, actual value: UPDATE_NEEDED
params.status: expected: UPDATE_NEEDED, actual value: UPDATING
 HMI call BasicCommunication.PolicyUpdate: Timeout: HMI call BasicCommunication.PolicyUpdate: Timeout expired
```
Please explain why this test is necessary and, if so, why it's omitted from the hmi_ptu test set. If it is not necessary please remove this test

Reply:
The idea of the test to check that the second PTU via HMI is successful after the failed first attempt, so this test is necessary for our feature.
Test is omitted in test set because of [defect](https://github.com/smartdevicelink/sdl_core/issues/3136) on develop branch, after fix is merged 016 and 017 scripts should be uncommented in test set.

Reply:
I see, thanks for clarifying. For now I would recommend adding a note in the hmi_ptu test set to explain why those tests are omitted. This note can be removed once the tests are uncommented